src/utils/debugLogger.ts

```diff
@@ -64,7 +64,7 @@ class DebugLogger {
 
 export const debugLogger = new DebugLogger();
 
-const kLogCount = 50;
+const kLogCount = 500;
```

Comment:
that's quite spammy, how about 150?

Reply:
We already set it to 500  before (https://github.com/microsoft/playwright/pull/6332) but I'm fine with 150 too :grin: